server/api/student/student.model.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var schoolYearSchema = new Schema({
```

Comment:
Lets keep the naming conventions consistent. I think the other files generated by angular-fullstack uses title case for Schema names.

`SchoolYearSchema`